--- lalinspiral-1.7.7/Makefile.am.orig	2015-03-27 05:14:39.000000000 +0100
+++ lalinspiral-1.7.7/Makefile.am	2017-07-02 21:10:16.094489089 +0200
@@ -23,11 +23,6 @@
 user_environment = \
 	set $(PACKAGE)_PREFIX $(prefix); \
 	set $(PACKAGE)_DATADIR $(pkgdatadir); \
-	prepend MANPATH $(mandir); \
-	prepend OCTAVE_PATH $(octexecdir); \
-	prepend PATH $(bindir); \
-	prepend PKG_CONFIG_PATH $(libdir)/pkgconfig; \
-	prepend PYTHONPATH $(pyexecdir) $(pythondir); \
 	$(END_OF_LIST)
 
 pkgconfigdir = $(libdir)/pkgconfig
